src/SamplesApp/SamplesApp.UWP/SamplesApp.UWP.csproj

```diff
@@ -117,6 +117,10 @@
     <Content Include="Properties\Default.rd.xml" />
   </ItemGroup>
   <ItemGroup>
+    <ProjectReference Include="..\..\Uno.UI.RuntimeTests\Uno.UI.RuntimeTests.csproj">
+      <Project>{028f3ee0-d51b-469a-a72c-c272585dcd40}</Project>
+      <Name>Uno.UI.RuntimeTests</Name>
+    </ProjectReference>
```

Comment:
This reference is not added to other heads, it's normal?

Reply:
Yes, it was already there for the others.